.gitignore

```diff
@@ -143,4 +143,4 @@ gradle-app.setting
 .classpath
 
 # End of https://www.toptal.com/developers/gitignore/api/netbeans,intellij,java,gradle,eclipse
-application/db/
+/db/
```

Comment:
did the structure change? please double check that this actually still covers the db stuff with the most recent version of develop.

Reply:
That's actually independent for everyone. I might as well just do `**/db/`